image_test/sbom/test_sbom_export.sh

```diff
@@ -0,0 +1,36 @@
+URL="http://metadata/computeMetadata/v1/instance/attributes"
+SBOM_FILE_NAME=$(curl -f -H Metadata-Flavor:Google ${URL}/sbom-file-name)
+OUTS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/outs-path)
```

Comment:
Also this file is being added by two PR I think.

Reply:
for now, the other PR is being ignored until this PR is merged.